pkg/daemon/ceph/client/filesystem.go

```diff
@@ -460,3 +460,31 @@ func listSubvolumesInGroup(context *clusterd.Context, clusterInfo *ClusterInfo,
 
 	return svs, nil
 }
+
+// SubVolumeSnapshot represents snapshot of a cephFS subvolume
+type SubVolumeSnapshot struct {
+	Name string `json:"name"`
+}
+
+// SubVolumeSnapshotList is the list of snapshots in a CephFS subvolume
+type SubVolumeSnapshotList []SubVolumeSnapshot
```

Comment:
`SubVolumeSnapshotList` to `SubVolumeSnapshots` we can remove `List` as its ending with plurals